relay-config/src/redis.rs

```diff
@@ -263,25 +263,23 @@ pub(super) fn build_redis_configs(
     configs: &RedisConfigs,
     cpu_concurrency: u32,
 ) -> RedisConfigsRef<'_> {
-    // Default `max_connections` for the `project_configs` client.
-    // In a unified config, this is used for all clients.
-    let project_configs_default_connections =
-        std::cmp::max(cpu_concurrency * 2, DEFAULT_MIN_MAX_CONNECTIONS);
+    // The default number of connections is twice the concurrency since we are using async Redis
+    // so a single thread might be doing more I/O concurrently.
+    let default_connections = std::cmp::max(cpu_concurrency * 2, DEFAULT_MIN_MAX_CONNECTIONS);
```

Comment:
For project configs we could use less, since they are completely separated from the processor. Maybe keep that one at `DEFAULT_MIN_MAX_CONNECTIONS`?

Tbh not sure right now if that makes sense.

Reply:
You mean that we could use less but keep the same minimum?

Reply:
So you would have both using `cpu_concurrency * 2` but have the minimum lower?